src/autoscaler/cf/client.go

```diff
@@ -122,7 +122,7 @@ func (c *cfClient) retrieveEndpoints() error {
 }
 
 func (c *cfClient) requestTokenGrant(formData *url.Values) error {
-	c.logger.Info("request-token-grant", lager.Data{"authUrl": c.authUrl, "form": *formData})
```

Comment:
I would prefer to have a function to replace passwords, credentials to be something like [hidden]. It can detect the type, and have several keywords mapping... 

I am hoping this function can handle map, db url,  http request/response ( to hide authorization header if you log the request headers)

Reply:
and I believe there are other places we need to hide passwords, credentials and tokens... I pointed you here just giving one example.

Another case is dburl which will have password in it.  

We need to go over the code and have a sanity check ...